applications/nftaa.md

```diff
@@ -0,0 +1,411 @@
+other assets, leading to the concept of NFTAA.
+
+One of the significant strengths of the Substrate framework is its ability to facilitate the creation of custom
+blockchain logic.
+Through forkless runtime upgrades, it is possible to enhance the functionality of individual modules.
+This flexibility is critical as runtime pallets operate close to the system’s core, offering virtually limitless
+modification options.
+Although the transaction weight is a limiting factor in terms of computing resources available for each transaction, it
+does not detract from the Substrate framework’s overall power and adaptability.
+
+Most Substrate pallets are designed based on a single account architecture, aligning with the NFTAA philosophy and
+facilitating smooth integration with existing infrastructure.
+By treating NFTAAs as conventional accounts, we can easily leverage the existing infrastructure for seamless
+communication.
+Additionally, NFTAAs can function as asset owners or participate in multi-signature schemes, further
+enhancing their utility and security.
+
+<img src="https://raw.githubusercontent.com/decenzio/NFTAA-Polkadot/286414435e0fdd51810c1093688daf67df59d010/proposal/prox.png" alt="Proposal Diagram" width="30%" height="30%">
+
+A significant feature of NFTAA is the ability to transfer ownership of such an account/asset to
+another owner with the real change of the private keys serving the NFTAA.
+For example, if Alice has an NFTAA in her wallet through which she stakes 100 DOT, Alice can sell the NFTAA to Bob,
+which causes Alice's wallet to no longer be able to manipulate the NFTAA,
+meaning, that Bob will have full access using his own private keys that only he knows.
+This gives NFTAA the unique ability to assess many complex use cases.
+
+#### Technical Details
+
+The implementation of the NFTAA concept within the Polkadot ecosystem can be approached in two primary ways.
+The first approach involves creating custom blockchain logic in the form of a pallet, written in Rust as a Substrate
+module.
+The second approach involves using smart contracts. For this proposal, we prefer the pallet approach due to its
+integration capabilities and performance benefits.
+
+The proposed state-of-the-art pallet is based on the _nfts_ and _utility_ pallets.
+The first step involves implementing and wrapping the NFT logic from the _nfts_ pallet into the _nftaa_ pallet.
+A collection in the _nfts_ runtime pallet is created to represent an equivalent of a smart contract instance.
+The _nftaa_ pallet marks a collection so that any token from this collection is treated as an account. After creating a
+mintable collection, the _nftaa_ special function must be called with the appropriate parameters, including _collection_id_
+and _attributes_.
+Attributes are crucial for enterprise use cases, allowing NFTAAs to be marked as entities such as companies or departments
+within a corporate structure.
```

Comment:
From our point of view, this is an opportunity to differentiate between already existing unique NFTs inside one company. To better manage and govern the orgs represented by NFTAAs or to add additional information to the NFT that is needed to implement the usecase we are writing about.

Reply:
We already have the identity pallet which is able to represent additional information linked to an account. Given that this is the established way of linking information to an account I suggest to re-evaluate this strategy. This NFT attribute approach would introduce unnecessary overhead to existing projects that aggregate data linked to accounts. e.g. Polkassembly.

Reply:
Thanks for this comment and I'm sure it can be used since NFTAA will be like an account so it shouldn't be a problem to join. However, in our case we were thinking more of the technical attributes but thus the detail will only be clear when implemented. We just wanted to have the "corridor" more open so we don't promise something which could be potentially improved.